api/src/main/java/org/openmrs/module/rwandahivflowsheet/impl/pih/TbEpisodeMapping.java

```diff
@@ -28,8 +29,8 @@ public DrugOrder getContinuationPhaseDrugOrder(){
 		Date initialDate = null;
 		for (DrugOrder dor : tbDrugOrders){
 			if (initialDate == null)
-				initialDate = dor.getStartDate();
-			else if (dor.getStartDate().getTime() > initialDate.getTime() && (
+				initialDate = dor.getEffectiveStartDate();
```

Comment:
`startDate` was replaced with `dateActivated` https://wiki.openmrs.org/x/yAgJAg

Reply:
With the concept of Urgency though, #getEffectiveStartDate seems to be the best option here. It returns #scheduledDate if Urgency is Scheduled or #dateActivated if Urgency is not Scheduled.

Reply:
Did you get a chance to look at what Nicholas Ingosi asked at the bottom of the link that i shared above?

Reply:
I hadn't seen it -sorry.
Here is the [fix](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/213f261e1a05d4051dc87143028a75f462b0b26d).